doc/source/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html

```diff
@@ -88,8 +94,17 @@
                     </p>
                     <p>
                         AWS CLI version 2, the latest major version of AWS CLI, is now stable and recommended for general use.
-                        To view this page for the AWS CLI version 2, click
-                        <a href="https://awscli.amazonaws.com/v2/documentation/api/latest/{{ pagename }}.html">here</a>.
+                        {% if pagename in deprecated_commands %}
+                            {% if deprecated_commands[pagename] %}
+                                This command is deprecated in AWS CLI version 2, use
+                                <a href="https://awscli.amazonaws.com/v2/documentation/api/latest/{{ deprecated_commands[pagename][0] }}.html">{{ deprecated_commands[pagename][1] }}</a> instead.
+                            {% else %}
+                                This command is deprecated in AWS CLI version 2,
```

Comment:
We should end this with a period instead of a comma. We should update the wording to be: `This command is deprecated and no longer available in AWS CLI version 2.` to make it clear that there is no other option.